services/meals/.env

```diff
@@ -0,0 +1,4 @@
+API_VERSION=v1
+API_PREFIX=/api
+PORT=3000
```

Comment:
question(blocking): to be honest, i don't know for sure, but maybe it is a good idea to change the port to `3001` to exclude possible future conflicts with cousins service